microcosm/config/api.py

```diff
@@ -0,0 +1,40 @@
+"""
+Configuration API.
+
+"""
+from microcosm.config.model import Configuration
+from microcosm.decorators import DEFAULTS
+
+
+def get_defaults(func):
+    """
+    Retrieve the defaults for a factory function.
+
+    """
+    return getattr(func, DEFAULTS, {})
+
+
+def configure(registry, metadata, loader):
```

Comment:
If you _squint_ you should notice that `configure` and `configure_scoped` have the same structure. My next step is to build better abstraction around these operations.